data/human/fleets.txt

```diff
@@ -1842,17 +1842,17 @@ fleet "Small Southern Pirates"
 	variant 2
 		"Hawk (Speedy)"
 	variant 2
-		"Hawk (Rocket)"
+		"Hawk"
 	variant 5
 		"Fury"
 	variant 4
-		"Fury (Bomber)"
+		"Fury"
 	variant 1
 		"Fury (Laser)"
 	variant 1
 		"Fury (Gatling)"
 	variant 1
-		"Fury (Missile)"
+		"Fury"
```

Comment:
Instead of having duplicated entries, I think it would be better to condense when the variant removal causes duplicates.